chromadb/server/fastapi/types.py

```diff
@@ -6,26 +6,24 @@
 )
 
 
-class AddEmbedding(BaseModel):  # type: ignore
```

Comment:
addressing a bunch of these errors:
`chromadb/server/fastapi/types.py:19: error: Unused "type: ignore" comment`